analysis/tests/src/expected/Dce.res.txt

```diff
@@ -1,3 +1,3 @@
 DCE src/Dce.res
-issues:243
+issues:249
```

Comment:
Would you add an issue reminding to solve this?
This change is pure noise w.r.t. the current PR.

Reply:
Done https://github.com/rescript-lang/rescript-vscode/issues/515